cmd/webhook/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"flag"
+
+	webhook "github.com/kubernetes-csi/external-snapshotter/v2/pkg/webhook"
```

Comment:
The v2 is part of the module name defined in go.mod. It's required to ensure go looks inside the directory instead of trying to download the webhook files from the internet.